src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -440,7 +440,7 @@ public class EnableWSManCredSSPCommand : WSManCredSSPCommandBase, IDisposable/*,
 
         //private const string DelegateComputerParam = "DelegateComputer";
         //private String[] delegatecomputer;
-        //private RuntimeDefinedParameterDictionary dyanmicParameters = new RuntimeDefinedParameterDictionary();
+        //private RuntimeDefinedParameterDictionary dynamicParameters = new RuntimeDefinedParameterDictionary();
```

Comment:
I researched the history and that's how the code was originally written in 2009 :)

I think it's safe to remove this commented-out code.